static/src/js/actions/__tests__/urlQuery.test.js

```diff
@@ -10,6 +10,121 @@ import * as urlQuery from '../urlQuery'
 
 const mockStore = configureMockStore([thunk])
 
+jest.mock('../../../../../portals/availablePortals.json', () => ({
```

Comment:
Is this same mock defined in 3 different places?

Reply:
Moved this to update the jest config